feature/gribi/otg_tests/mpls_in_udp_ipsec/README.md

```diff
@@ -0,0 +1,200 @@
+              src_ip: "outer_ipv6_src"
+              dst_ip: "outer_ipv6_dst_A"
+              dst_udp_port: "outer_dst_udp_port"
+              ip_ttl: "outer_ip-ttl"
+              dscp: "outer_dscp"
+              key_chain_name: "ipsec_keychain"
+            }
+          }
+        }
+      }
+      #
+      # entries used for "group_B"
+      ipv6_unicast {
+        ipv6_entry {
+          prefix: "inner_ipv6_dst_B"
+          next_hop_group: 200
+        }
+      }
+      ipv4_unicast {
+        ipv4_entry {
+          prefix: "ipv4_inner_dst_B"
+          next_hop_group: 200
+        }
+      }
+      next_hop_groups {
+        next_hop_group {
+          next_hop_group_id: "nhg_A"  # new OC path /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          id: 200
+          next_hops {            # reference to a next-hop
+            next_hop: {
+              index: 200
+            }
+          }
+        }
+      }
+      next_hops {
+        next_hop {
+          index: 200
+          network_instance: "group_B"
+          encap-headers {
+            encap-header {
+              index: 1
+              type : OPENCONFIG_AFT_TYPES:MPLS
+              mpls {
+                pushed_mpls_label_stack: [200,]
+              }
+            }
+          }
+          encap-headers {
+            encap-header {
+              index: 2
+              type: OPENCONFIG_AFT_TYPES:UDP
+              udp {
+                src_ip: "outer_ipv6_src"
+                dst_ip: "outer_ipv6_dst_B"
+                dst_udp_port: "outer_dst_udp_port"
+                ip_ttl: "outer_ip-ttl"
+                dscp: "outer_dscp"
+                key_chain_name: "ipsec_keychain"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+
+  # qos classifier config
+  /qos/classifiers/classifier/config/name:
+  /qos/classifiers/classifier/terms/term/config/id:
+  #/qos/classifiers/classifier/terms/term/conditions/next-hop-group/config/name: # TODO: new OC leaf to be added
+
+  # afts next-hop counters
+  /network-instances/network-instance/afts/next-hops/next-hop/state/counters/packets-forwarded:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/counters/octets-forwarded:
+
+  # afts state paths set via gRIBI
+  # TODO: https://github.com/openconfig/public/pull/1153
+  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/src-ip:
+  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dst-ip:
+  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/ip-ttl:
+  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dst-udp-port:
+  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dscp:
+  #/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/next-hop-group-id:
```

Comment:
these paths shuold be updated to match the newer #1153 model with the encap-headers container